praw/models/helpers.py

```diff
@@ -23,6 +23,35 @@ def __call__(self, id):  # pylint: disable=invalid-name,redefined-builtin
         """
         return LiveThread(self._reddit, id=id)
 
+    def info(self, ids):
+        """Fetch information about each live thread in ``ids``.
+
+        :param ids: A list of IDs for live threads.
+        :returns: A generator that yields :class:`.LiveThread` instances
```

Comment:
Is this returning a generator, or a direct list?

Does this endpoint have a 100 item limit? If so, perhaps it needs to be batched in groups of 100.